HW_3/zd2.py

```diff
@@ -0,0 +1,15 @@
+#Задача 28: Напишите рекурсивную функцию sum(a, b), возвращающую сумму двух целых неотрицательных чисел.
+#Из всех арифметических операций допускаются только +1 и -1. Также нельзя использовать циклы.
+
+a = int(input("Введите первое неотрицительное число "))
+b = int(input("Введите второе неотрицательно число "))
+
+
+def recursive_sum(a, b):
+    if a == 0:
+        return b
+    else:
+        return recursive_sum(a - 1, b + 1)
```

Comment:
Блок кода `else` не нужен, ведь `return` завершает работу функции.
```suggestion
    return recursive_sum(a - 1, b + 1)
```